

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>MGI CleanPlex SOP &mdash; McGill COVID-19 Sequencing SOP  documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
        <script src="../_static/jquery.js"></script>
        <script src="../_static/underscore.js"></script>
        <script src="../_static/doctools.js"></script>
        <script src="../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Illumina ARTIC SOP" href="Illumina_overview.html" />
    <link rel="prev" title="Nanopore ARTIC-Nanopolish SOP" href="ONT_overview.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../index.html" class="icon icon-home"> McGill COVID-19 Sequencing SOP
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Contents:</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="about.html">Québec SARS-CoV-2 Sequencing Project</a></li>
<li class="toctree-l1"><a class="reference internal" href="ONT_overview.html">Nanopore ARTIC-Nanopolish SOP</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">MGI CleanPlex SOP</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#graphical-summary">Graphical summary</a></li>
<li class="toctree-l2"><a class="reference internal" href="#description">Description</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#steps">Steps</a></li>
<li class="toctree-l3"><a class="reference internal" href="#reference-genome-and-software-versions">Reference Genome and Software Versions</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Illumina_overview.html">Illumina ARTIC SOP</a></li>
<li class="toctree-l1"><a class="reference internal" href="../COVID19_BioBank/overview.html">COVID-19 BioBank SOP</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../index.html">McGill COVID-19 Sequencing SOP</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../index.html">Docs</a> &raquo;</li>
        
      <li>MGI CleanPlex SOP</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../_sources/SARS_CoV2_Sequencing/MGI_overview.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="mgi-cleanplex-sop">
<h1>MGI CleanPlex SOP<a class="headerlink" href="#mgi-cleanplex-sop" title="Permalink to this headline">¶</a></h1>
<div class="section" id="graphical-summary">
<h2>Graphical summary<a class="headerlink" href="#graphical-summary" title="Permalink to this headline">¶</a></h2>
<a class="reference internal image-reference" href="../_images/MGI_CleanplexPipeline.png"><img alt="../_images/MGI_CleanplexPipeline.png" class="align-center" src="../_images/MGI_CleanplexPipeline.png" style="width: 666.6px; height: 375.0px;" /></a>
</div>
<div class="section" id="description">
<h2>Description<a class="headerlink" href="#description" title="Permalink to this headline">¶</a></h2>
<p>The MGI analysis pipeline is based on the steps suggested by <em>Paragon Genomics</em>, the manufacturer of the
<code class="docutils literal notranslate"><span class="pre">CleanPlex</span></code> amplicon panel, with important modifications to improve performance.</p>
<div class="section" id="steps">
<h3>Steps<a class="headerlink" href="#steps" title="Permalink to this headline">¶</a></h3>
<ol class="arabic simple">
<li><p><strong>Align read with</strong> <code class="docutils literal notranslate"><span class="pre">bwa</span> <span class="pre">mem</span></code> <strong>, filter out human reads on the bam file and convert to fastq</strong></p></li>
</ol>
<p>1.1 Raw reads are aligned with <code class="docutils literal notranslate"><span class="pre">bwa</span> <span class="pre">mem</span></code> and sorted by position with <code class="docutils literal notranslate"><span class="pre">sambamba</span></code> for merics</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>bwa mem  \
  -K 100000000 -v 3 -t 8 -Y \
  -R &#39;@RG\tID:${SAMPLE}\tSM:${SAMPLE}\tLB:${LIBRARY}\tPU:run1_2\tCN:McGill University and Genome Quebec Innovation Centre\tPL:MGI&#39; \
  Coronavirinae.SARS-CoV-2/genome/bwa_index/Hybrid.SARS-CoV-2.GRCh38.fa \
  data/${SAMPLE}/${SAMPLE}.pair1.fastq.gz \
  data/${SAMPLE}/${SAMPLE}.pair2.fastq.gz | \
sambamba view -S -f bam \
  /dev/stdin | \
sambamba sort  \
  /dev/stdin \
  --out alignment/${SAMPLE}/${SAMPLE}.hybrid.sorted.bam
</pre></div>
</div>
<p>1.2 Human mapped reads with a mapping quality &gt; 0 are filtered out and then sorted by name with <code class="docutils literal notranslate"><span class="pre">sambamba</span></code></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>sambamba view \
  -f bam -F &quot;not (ref_name =~ /chr*/ and mapping_quality &gt; 0)&quot; \
  host_removal/${SAMPLE}/${SAMPLE}_RAG936A8-12.hybrid.sorted.bam | \
sambamba sort \
  -N \
  /dev/stdin \
  --out host_removal/${SAMPLE}/${SAMPLE}.host_removed.nsorted.bam
</pre></div>
</div>
<p>1.3 Conversion of the bam filtered out and sorted by name into fastqs with <code class="docutils literal notranslate"><span class="pre">samtools</span></code></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>samtools bam2fq \
  -@ 10 \
  -n \
  -1 host_removal/${SAMPLE}/${SAMPLE}.host_removed.pair1.fastq.gz \
  -2 host_removal/${SAMPLE}/${SAMPLE}.host_removed.pair2.fastq.gz \
  -0 host_removal/${SAMPLE}/${SAMPLE}.host_removed.other.fastq.gz \
  -s host_removal/${SAMPLE}/${SAMPLE}.host_removed.single.fastq.gz \
  host_removal/${SAMPLE}/${SAMPLE}.host_removed.nsorted.bam
</pre></div>
</div>
<ol class="arabic simple" start="2">
<li><p><strong>Trim sequencing adaptor with</strong> <code class="docutils literal notranslate"><span class="pre">cutadapt</span></code></p></li>
</ol>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>cutadapt -g GAACGACATGGCTACGATCCGACTT \
  -G GACCGCTTGGCCTCCGACTT \
  -a AAGTCGGAGGCCAAGCGGTC \
  -A AAGTCGGATCGTAGCCATGTCGTTC \
  -j 5 \
  -e 0.1 -O 9 -m 20 -n 2 --quality-cutoff 33 \
  -o cleaned_raw_reads/${SAMPLE}/${SAMPLE}.trim.pair1.fastq.gz \
  -p cleaned_raw_reads/${SAMPLE}/${SAMPLE}.trim.pair2.fastq.gz  \
  host_removal/${SAMPLE}/${SAMPLE}.host_removed.pair1.fastq.gz  \
  host_removal/${SAMPLE}/${SAMPLE}.host_removed.pair2.fastq.gz
</pre></div>
</div>
<ol class="arabic simple" start="3">
<li><p><strong>Align read with</strong> <code class="docutils literal notranslate"><span class="pre">bwa</span> <span class="pre">mem</span></code> <strong>and filter the bam file</strong></p></li>
</ol>
<p>3.1 Multimapping of the reads with <code class="docutils literal notranslate"><span class="pre">bwa</span> <span class="pre">mem</span></code> and sort with <code class="docutils literal notranslate"><span class="pre">sambamba</span></code></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>bwa mem  \
  -K 100000000 -v 3 -t 8 -Y \
  -R &#39;@RG\tID:${SAMPLE}\tSM:${SAMPLE}\tLB:${LIBRARY}\tPU:run1_2\tCN:McGill University and Genome Quebec Innovation Centre\tPL:MGI&#39; \
  Coronavirinae.SARS-CoV-2/genome/bwa_index/Coronavirinae.SARS-CoV-2.fa \
  trim/${SAMPLE}/${SAMPLE}.trim.pair1.fastq.gz \
  trim/${SAMPLE}/${SAMPLE}.trim.pair2.fastq.gz | \
sambamba view -S -f bam \
  /dev/stdin | \
sambamba sort  \
  /dev/stdin \
  --out alignment/${SAMPLE}/${SAMPLE}.sorted.bam &amp;&amp; \
sambamba index  \
  alignment/${SAMPLE}/${SAMPLE}.sorted.bam \
  alignment/${SAMPLE}/${SAMPLE}.sorted.bam.bai
</pre></div>
</div>
<p>3.2 Mapped reads are filtered with <code class="docutils literal notranslate"><span class="pre">sambamba</span></code> and <code class="docutils literal notranslate"><span class="pre">awk</span></code></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>sambamba view -h \
  alignment/${SAMPLE}/${SAMPLE}.sorted.bam | \
awk &#39;substr($0,1,1)==&quot;@&quot; || ($9 &gt;= 60 &amp;&amp; $9 &lt;= 300) || ($9 &lt;= -60 &amp;&amp; $9 &gt;= -300)&#39; | \
sambamba view -S -f bam -F &quot;not failed_quality_control and not unmapped and not secondary_alignment and mapping_quality &gt; 0 and not supplementary and not (((mate_is_reverse_strand and reverse_strand) or (not reverse_strand and not mate_is_reverse_strand)))&quot; \
  /dev/stdin \
  -o alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.bam
</pre></div>
</div>
<ol class="arabic simple" start="4">
<li><p><strong>Trim Amplicon primers with</strong> <code class="docutils literal notranslate"><span class="pre">ivar</span></code> <strong>and</strong> <code class="docutils literal notranslate"><span class="pre">samtools</span></code></p></li>
</ol>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>ivar trim -i alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.bam \
  -p alignment/${SAMPLE}/${SAMPLE}.primerTrim \
  -b Coronavirinae.SARS-CoV-2/PGD99s.ampInfo_+-.bed \
  -m 30 -q 20 -s 4 -e &amp;&amp; \
sambamba sort  \
  alignment/${SAMPLE}/${SAMPLE}.primerTrim.bam \
  --out alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.bam
</pre></div>
</div>
<ol class="arabic simple" start="5">
<li><p><strong>Call variants with</strong> <code class="docutils literal notranslate"><span class="pre">ivar</span></code> <strong>and</strong> <code class="docutils literal notranslate"><span class="pre">samtools</span></code></p></li>
</ol>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>samtools mpileup -A -d 1000000 -B -Q 0 \
  -l $MUGQIC_INSTALL_HOME_DEV/genomes/species/Coronavirinae.SARS-CoV-2/SARSCoV2.ampInsert.bed \
  --reference Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  -f Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.bam | \
ivar variants -p alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim \
  -r Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  -m 10 -q 20 -t 0.25 &amp;&amp; \
ivar_variants_to_vcf.py alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.tsv alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.vcf &amp;&amp; \
bgzip -cf \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.vcf &gt; \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.vcf.gz &amp;&amp; \
tabix -pvcf alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.vcf.gz
</pre></div>
</div>
<ol class="arabic simple" start="6">
<li><p><strong>Annotate the variants with</strong> <code class="docutils literal notranslate"><span class="pre">SnpEff</span></code></p></li>
</ol>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>java -XX:ParallelGCThreads=1 -Dsamjdk.buffer_size=4194304 -Xmx20G \
  -jar $SNPEFF_HOME/snpEff.jar ann \
  -v -c $SNPEFF_HOME/snpEff.config \
  -s metrics/dna/${SAMPLE}/snpeff_metrics/${SAMPLE}.snpEff.html \
  MN908947.3 \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.vcf.gz &gt; \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.annotate.vcf &amp;&amp; \
bgzip -cf \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.annotate.vcf &gt; \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.annotate.vcf.gz &amp;&amp; \
tabix -pvcf alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.annotate.vcf.gz
</pre></div>
</div>
<ol class="arabic simple" start="7">
<li><p><strong>Generate consensus FASTA and rename header with</strong> <code class="docutils literal notranslate"><span class="pre">ivar</span></code> <strong>and</strong> <code class="docutils literal notranslate"><span class="pre">samtools</span></code></p></li>
</ol>
<p>7.1 Consensus generation with** <code class="docutils literal notranslate"><span class="pre">ivar</span></code> <strong>and</strong> <code class="docutils literal notranslate"><span class="pre">samtools</span></code></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>samtools mpileup -aa -A -d 600000 -Q 0 \
  -f Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.bam | \
ivar consensus -p alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.consensus -q 20 -t 0.75 -m 10
</pre></div>
</div>
<p>7.2 Renaming of header for GISAID submission</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>awk &#39;/^&gt;/{print &quot;&gt;Canada/Qc-${SAMPLE}_${PLATE}/${YEAR} seq_method:MGI_CleanPlex|assemb_method:ivar|snv_call_method:ivar&quot;; next}{print}&#39; &lt; \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.consensus.fa &gt; \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.consensus.gisaid_renamed.fa
</pre></div>
</div>
<ol class="arabic simple" start="8">
<li><p><strong>Compute metrics</strong></p></li>
</ol>
<p>A series of scripts compute several metrics derived from the output of the analysis above. Here are the full set
of commands used to generate these metrics.:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>#alignment and insert size metrics
gatk --java-options &quot;-XX:ParallelGCThreads=1 -Dsamjdk.buffer_size=4194304 -Xmx20G&quot; \
  CollectMultipleMetrics \
  --PROGRAM=CollectAlignmentSummaryMetrics \
  --PROGRAM=CollectInsertSizeMetrics \
  --VALIDATION_STRINGENCY=SILENT \
  --REFERENCE_SEQUENCE=Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  --INPUT=alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  --OUTPUT=metrics/dna/${SAMPLE}/picard_metrics/${SAMPLE}.all.metrics \
  --MAX_RECORDS_IN_RAM=1000000

#OxoG metrics
gatk --java-options &quot;-XX:ParallelGCThreads=1 -Dsamjdk.buffer_size=4194304 -Xmx20G&quot; \
  CollectOxoGMetrics \
  --VALIDATION_STRINGENCY=SILENT  \
  --INPUT=alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  --OUTPUT=metrics/dna/${SAMPLE}/picard_metrics/${SAMPLE}.oxog_metrics.txt \
  --REFERENCE_SEQUENCE=Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  --MAX_RECORDS_IN_RAM=4000000

#GC biais metrics
gatk --java-options &quot;-XX:ParallelGCThreads=1 -Dsamjdk.buffer_size=4194304 -Xmx20G&quot; \
  CollectGcBiasMetrics \
  --VALIDATION_STRINGENCY=SILENT \
  --ALSO_IGNORE_DUPLICATES=TRUE \
  --INPUT=alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  --OUTPUT=metrics/dna/${SAMPLE}/picard_metrics/${SAMPLE}.qcbias_metrics.txt \
  --CHART=metrics/dna/${SAMPLE}/picard_metrics/${SAMPLE}.qcbias_metrics.pdf \
  --SUMMARY_OUTPUT=metrics/dna/${SAMPLE}/picard_metrics/${SAMPLE}.qcbias_summary_metrics.txt \
  --REFERENCE_SEQUENCE=Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  --MAX_RECORDS_IN_RAM=4000000

#qualimap metrics
qualimap bamqc -nt 10 \
  -bam alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam -outdir metrics/dna/${SAMPLE}/qualimap \
  --java-mem-size=55G

#flagstats metrics primerTrimed
sambamba flagstat  \
  alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  &gt; metrics/dna/${SAMPLE}/flagstat/${SAMPLE}.sorted.primerTrim.flagstat

#bedGraph generation primerTrimmed
bedtools genomecov -bga \
  -ibam alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam &gt; alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.BedGraph

#hs metrics
gatk --java-options &quot;-XX:ParallelGCThreads=1 -Dsamjdk.buffer_size=4194304 -Xmx20G&quot; \
  CollectHsMetrics \
  --INPUT=alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  --OUTPUT=alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.onTarget.tsv \
  --BAIT_INTERVALS=Coronavirinae.SARS-CoV-2/SARSCoV2.ampInsert.interval_list \
  --TARGET_INTERVALS=SARSCoV2.ampInsert.interval_list \
  --REFERENCE_SEQUENCE=Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa

#depth of coverage
java -XX:ParallelGCThreads=2 -Xmx8000M -jar $GATK_JAR \
  --analysis_type DepthOfCoverage --omitDepthOutputAtEachBase --logging_level ERROR \
  --reference_sequence Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  --input_file alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  --out alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.all.coverage \
  --intervals Coronavirinae.SARS-CoV-2/SARSCoV2.ampInsert.bed \
  --summaryCoverageThreshold 10 \
  --summaryCoverageThreshold 25 \
  --summaryCoverageThreshold 50 \
  --summaryCoverageThreshold 75 \
  --summaryCoverageThreshold 100 \
  --summaryCoverageThreshold 500 \
  --start 1 --stop 500 \
  --nBins 499 \
  --downsampling_type NONE

java -XX:ParallelGCThreads=1 -Dsamjdk.buffer_size=4194304 -Xmx31G -jar $BVATOOLS_JAR \
  depthofcoverage --gc --maxDepth 1001 --summaryCoverageThresholds 10,25,50,75,100,500,1000 --minMappingQuality 15 --minBaseQuality 15 --ommitN \
  --threads 8 \
  --ref Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  --intervals Coronavirinae.SARS-CoV-2/SARSCoV2.ampInsert.bed \
  --bam alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  &gt; alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.coverage.tsv

#generate tdf file
java -Xmx8G  -Djava.awt.headless=true -jar $IGVTOOLS_JAR count -f min,max,mean -w 25 \
  alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.bam \
  alignment/${SAMPLE}/${SAMPLE}.sorted.primerTrim.tdf \
  Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa.fai

#generate quast metrics
quast.py -r Coronavirinae.SARS-CoV-2/genome/Coronavirinae.SARS-CoV-2.fa \
  --output-dir metrics/dna/${SAMPLE}/quast_metrics \
  --threads 5 \
  alignment/${SAMPLE}/${SAMPLE}.sorted.filtered.primerTrim.consensus.gisaid_renamed.fa
</pre></div>
</div>
</div>
<div class="section" id="reference-genome-and-software-versions">
<h3>Reference Genome and Software Versions<a class="headerlink" href="#reference-genome-and-software-versions" title="Permalink to this headline">¶</a></h3>
<p><strong>Reference Genome:</strong> Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1 (GenBank MN908947.3)</p>
<p><strong>Software versions</strong></p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">bedtools</span> <span class="n">v2</span><span class="o">.</span><span class="mf">29.2</span>
<span class="n">bvatools</span> <span class="n">v1</span><span class="o">.</span><span class="mi">6</span>
<span class="n">bwa</span> <span class="n">v0</span><span class="o">.</span><span class="mf">7.17</span>
<span class="n">cutadapt</span> <span class="n">v2</span><span class="o">.</span><span class="mi">10</span>
<span class="n">GATK</span> <span class="n">v4</span><span class="o">.</span><span class="mf">1.2</span> <span class="ow">and</span> <span class="n">v3</span><span class="o">.</span><span class="mi">8</span>
<span class="n">igvtools</span> <span class="n">v2</span><span class="o">.</span><span class="mf">3.67</span>
<span class="n">ivar</span> <span class="n">v1</span><span class="o">.</span><span class="mf">2.2</span>
<span class="n">qualimap</span> <span class="n">v2</span><span class="o">.</span><span class="mf">2.1</span>
<span class="n">quast</span> <span class="n">v5</span><span class="o">.</span><span class="mf">0.2</span>
<span class="n">sambamba</span> <span class="n">v0</span><span class="o">.</span><span class="mf">7.0</span>
<span class="n">samtools</span> <span class="n">v1</span><span class="o">.</span><span class="mi">9</span>
<span class="n">snpEff</span> <span class="n">v4</span><span class="o">.</span><span class="mi">5</span><span class="n">covid19</span>
</pre></div>
</div>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="Illumina_overview.html" class="btn btn-neutral float-right" title="Illumina ARTIC SOP" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="ONT_overview.html" class="btn btn-neutral float-left" title="Nanopore ARTIC-Nanopolish SOP" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2020, C3G

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>